contrib/changelog.hbs

```diff
@@ -1,5 +1,7 @@
 {{header}}
 {{toHeader header "-"}}
 
+Date: {{date}}
```

Comment:
This should be an reST fied list:
http://docutils.sourceforge.net/docs/user/rst/quickref.html#field-lists

For example, note the difference in display here:
http://rst.ninjs.org/#RGF0ZTogTWFyY2ggMjMsIDIwMTgKCjpEYXRlOiBNYXIgMjMsIDIwMTg=

Reply:
Hi @agjohnson , I have made the changes. Do guide me if anything more has to be done :)